### PR TITLE
Changelog Updates

### PR DESCRIPTION
**Pull request by rcoringrato-sfdc**
### What does this PR do?
- Change log updates for v48.18.0

### CODE REVIEW

**Comment by rcoringrato-sfdc** [comment]
@hbc-bindu Can you review the changes when you get a chance? Thanks.

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+
+## Fixed
+
+#### salesforcedx-vscode-lwc
+
+- Fix LWC preview on Windows when running `SFDX: Preview Component Locally` ([PR #2212](https://github.com/forcedotcom/salesforcedx-vscode/pull/2212), [Issue #1903](https://github.com/forcedotcom/salesforcedx-vscode/issues/1903))
+
+#### salesforcedx-utils-vscode
+
+- Correctly surface errors for failed CLI commands ([PR #2216](https://github.com/forcedotcom/salesforcedx-vscode/pull/2216), [Issue #2160](https://github.com/forcedotcom/salesforcedx-vscode/issues/2160), [Issue #2005](https://github.com/forcedotcom/salesforcedx-vscode/issues/2005))
```
Comment (hbc-bindu) [inline packages/salesforcedx-vscode/CHANGELOG.md:11]: @rcoringrato-sfdc , consider rephrasing:
Display an error message in Apex Interactive Debugger when a user doesn’t have the required permission.

Comment (lcampos) [reply packages/salesforcedx-vscode/CHANGELOG.md:11]: @hbc-bindu @rcoringrato-sfdc the fix is not exclusive to errors caused by a user not having certain perms. We could rephrase it to `Display error messages in Apex Interactive Debugger ...`

Comment (lcampos) [reply packages/salesforcedx-vscode/CHANGELOG.md:11]: I take that back, the fix is not exclusive to the interactive debugger, that is just the place where we first noticed the problem. That code we fixed is used in other areas likes org authentication, isv debugger setup, etc.
